src/transformers/models/mobilebert/modeling_tf_mobilebert.py

```diff
@@ -936,6 +921,7 @@ def __init__(self, config, *inputs, **kwargs):
         output_type=TFBaseModelOutputWithPooling,
         config_class=_CONFIG_FOR_DOC,
     )
+    @unpack_inputs
```

Comment:
Sorry to be picky about this -- other models have been adding `@unpack_inputs` as the top decorator, can I ask you to move it to the top? That way we'll get consistent code :D

Reply:
Okay, I will move it